src/components/film-list-item/film-list-item.js

```diff
@@ -0,0 +1,19 @@
+import './film-list-item.css'
+
+function FilmListItem ({img, name, year, genres}) {
+
+    return (
+        <>
+            <div className={'film_list_item_container'}>
+                <img className={'film_list_item_img'} src={img}/>
+                <div class={'film_list_item_info'}>
+                <div className={'film_list_item_name'}>{name}</div>
+                <div className={'film_list_item_year'}>{year}</div>
+                </div>
+                <div className={'film_list_item_genres'}>{genres}</div>
+            </div>
+        </>
+    )
+}
```

Comment:
```suggestion
const FilmListItem = ({img, name, year, genres}) => 
    (<div className={'film_item_container'}>
                <img className='film_item_img' src={img}/>
                <div class='film_item_info'>
                <div className='film_item_name'>{name}</div>
                <div className='film_item_year'>{year}</div>
                </div>
                <div className='film_item_genres'>{genres}</div>
            </div>);
```
1.  Better use arrow functions when possible. https://javascript.plainenglish.io/5-ways-to-optimize-your-functional-react-components-cb3cf6c7bd68
2. If the element className doesn't change it will be better to use only single quotes.
3. Too long the class names.

Reply:
Done